herbert_core/utilities/classes/@unique_references_container/unique_references_container.m

```diff
@@ -444,6 +490,54 @@
             nuix = numel(self.idx_);
         end
 
+        function [self, nuix] = add_copies_(self,obj,n)
+            %ADD_COPIES - add a single object obj at the end of the container
+            % multiple times
+            %
+            % Input:
+            % ------
+            % obj - the single object to be added. This must not be a
+            %       unique container or an array of size.1 or cell.
+            % n   - the number of copies to add
+            %
+            % Output
+            % ------
+            % self - the revised container with the additional indices for
+            %        the added objects
+            % nuix - the range of added non-unique indices
+            
+            if isempty(self.stored_baseclass_)
+                error('HERBERT:unique_references_container:incomplete_setup', ...
+                    'stored baseclass unset');
+            end
+            if ~isa(obj,self.stored_baseclass_)
+                warning('HERBERT:unique_references_container:invalid_argument', ...
+                    'not correct stored base class; object was not added');
+                nuix = 0;
+                return;
+            end
+            if isempty(self.global_name_)
+                error('HERBERT:unique_references_container:incomplete_setup', ...
+                    'global name unset');
```

Comment:
Is this likely to mean anything to users or is this an internal error?

Reply:
Both. In the current usage of object duplication removal, it will be internal; however if the user wants to start rolling their own containers, this will catch errors of implementation for them